src/test/java/com/cloudant/tests/DatabaseTest.java

```diff
@@ -71,6 +72,22 @@ public void permissions() {
 
     }
 
+    /**
+     * Test that when called against a DB that is not a Cloudant service
+     * an UnsupportedOperationException is thrown
+     */
+    @Test
+    public void testPermissionsException() {
+        try {
+            Map<String, EnumSet<Permissions>> userPerms = db.getPermissions();
+            //ignore the test in cases where we don't get an exception
+            Assume.assumeTrue(false);
+        }catch(UnsupportedOperationException e){

```

Comment:
Not with an Assume in place, because Assume throws an `AssumptionViolatedException` which then causes the test to fail because it wasn't the expected exception. I will be able to used `expected=` however, if we get rid of the Assume with more categories as suggested by @tomblench.